doc/project_services/irker.md

```diff
@@ -0,0 +1,43 @@
+# Irker IRC Gateway
+
+GitLab provides a way to push update messages to an Irker server. When
+configured, pushes to a project will trigger the service to send data directly
+to the to the Irker server.
+
+See th eproject homepage for further info: http://www.catb.org/esr/irker/
+
+## Needed setup
+
+You will first need an Irker daemon. You can download the Irker code from its
+gitorious repository on https://gitorious.org/irker: `git clone
+git@gitorious.org:irker/irker.git`. Once you have downloaded the code, you can
+run the python script named `irkerd`. This script is the gateway script, it acts
+both as an IRC client, for sending messages to an IRC server obviously, and as a
+TCP server, for receiving messages from the GitLab service.
+
+If the Irker server runs on the same machine, you are done. If not, you will
+need to follow the firsts steps of the next section.
+
+## Optional setup
+
+In the 'production' section of the config/gitlab.yml file, you can add an
+'irker' subsection with the following keys:
+- **server_ip** (defaults to `localhost`) : the server IP address where the
+`irkerd` daemon runs;
+- **server_port** (defaults to `6659`) : the server port of the `irkerd` daemon;
+- **max_channels** (defaults to `3`) : the maximum number of recipients the
+client is authorized to join, per project;
+- **default_irc_uri** (no default) : if this option is set, it has to be of the
+format `irc[s]://domain.name` and will be prepend to each and every channel
+provided be the user which is not a full URI.
+
+If the Irker server and the GitLab application do not run on the same host, you
+will **need** to setup at least the **server_ip** option.
```

Comment:
Is it better to put hardcoded values for this pull request and see for these settings in another one? When I add settings, it will enlarge the commit, I'm not sure how you like very big commits(?).